packages/helpers/src/media.js

```diff
@@ -0,0 +1,53 @@
+import tokens from '@gympass/yoga-tokens';
+import { css } from 'styled-components';
+
+import hide from './hide';
+
+const { breakpoints } = tokens;
+
+const media = {
+  not: {},
+};
+
+export const not = isNot => (isNot ? 'not all and ' : '');
+
+const getRange = (width, range) => {
+  if (Array.isArray(width)) {
+    const [min, max] = width;
+
+    const {
+      [min]: { width: minBreakpoint },
+      [max]: { width: maxBreakpoint },
+    } = breakpoints;
```

Comment:
Do we need to `try/catch` this block to return a better error? Invalid parameters, for example.

```js
media.between('sm', 'lgg') // Misspelling
```

Reply:
Done!